### DISCUSSION TITLE
Ryzen 7000 Series APU?

### DISCUSSION THREAD

**Discussion by aurelilys** [ChefKissInc/NootedRed#71] [Q&A] [answered]
Hello,

Do you plan to support new ryzen 7000 series integrated graphic?

**Comment by AlphaNecron** [comment]
It's basically impossible.

**Comment by VisualEhrmanntraut** [comment, marked as answer]
@aurelilys Ryzen 7000 is RDNA 2, and that's supported by macOS, therefore it should be relatively easy to add support for it. However we currently are preoccupied by other tasks such as Catalina and Ventura support, among some other issues. We are also lacking testers for this.
In short, it's possible, and we do plan to support it, as long as we can get at least one tester for it.
I hope that answers your question

**Reply by AlphaNecron** [reply]
Uh I thought 6xxx and 7xxx iGPU are not supported?

**Reply by VisualEhrmanntraut** [reply]
Do you know how to read

**Reply by AlphaNecron** [reply]
Okay.

**Reply by b00t0x** [reply]
The Ryzen 7000 series has multiple combinations of CPU cores and GPU cores, is my understanding below correct?

Ryzen 7045 = Ryzen 7000 Desktop (Zen4/RDNA2) -> theoretically possible
Ryzen 7040 (Zen4/RDNA3) -> never works
Ryzen 7035 = Ryzen 6000 APU (Zen3+/RDNA2) -> theoretically possible
Ryzen 7030 = Ryzen 5000 APU (Zen3/Vega) -> already working
Ryzen 7020 = Steam Deck APU (Zen2/RDNA2) -> theoretically possible

**Reply by VisualEhrmanntraut** [reply]
Looks like it

**Reply by VisualEhrmanntraut** [reply]
> > Looks like it
> 
> Well if you ever get the time & resources to do it then consider us willing testers here :)
> 
> Albeit I am on a Steamdeck… which has had some success on the CPU side of things but people keep saying that the iGPU won’t be supported since it’s an AMD iGPU and not intel. Judging by the work here though it seems like there’s hope afterall for a Steamtosh Hackdeck! 😄

Van Gogh is just RDNA 2, and the HWLibs kext also has code for Van Gogh specifically
So it's likely possible too

**Reply by MumblingGuac** [reply]
If you guys need any testers of 6800H, which is Zen3+/RDNA2, I am very glad to be the one!

**Reply by VisualEhrmanntraut** [reply]
Hey @MumblingGuac. If you want to become a tester, you'll have to join our Telegram group located at [REDACTED]

**Reply by ghost** [reply]
@VisualEhrmanntraut i would actually like to be a tester for this if you have made some progress and want someone to test it
do you have a discord maybe?

**Reply by VisualEhrmanntraut** [reply]
@ThijsGaming I don't have anything for you to test. And I don't really use discord

**Reply by ghost** [reply]
ok i understand
my phone that had telegram broke and telegram desktop (on linux from flathub) wont send the sms code on an phone with the same sim
so how should i chat with you (outside of github)

**Reply by MurtadhaS** [reply]
I can test it on ryzen 7600 if you want.

**Reply by AngeloAvv** [reply]
@VisualEhrmanntraut I can test on a 7900X, where can I join the telegram group?

**Reply by VisualEhrmanntraut** [reply]
> I don't have anything for you to test

**Reply by MurtadhaS** [reply]
ok.

**Reply by iSoldLeo** [reply]
Very need it!!! Please!

**Comment by huytruong733** [comment]
not set vram 1gb or 2gb from bios default 512mb vram working for kext cpu 5980hs

**Reply by AlphaNecron** [reply]
Just set to 1GB or 2GB, 512MB is not recommended and may cause instabilities or other issues.

**Reply by AlphaNecron** [reply]
Just set to 1GB or 2GB, 512MB is not recommended and may cause instabilities or other issues.

**Reply by huytruong733** [reply]
no bios opition set vram size :))

**Reply by AlphaNecron** [reply]
There are only some systems with builtin option to set iGPU VRAM size in BIOS. For others, you have to use UMAF from https://github.com/DavidS95/Smokeless_UMAF.
